drivers/remote/driver.go

```diff
@@ -28,10 +28,25 @@ func newDriver(name string, client *plugins.Client) driverapi.Driver {
 // plugin is activated.
 func Init(dc driverapi.DriverCallback) error {
 	plugins.Handle(driverapi.NetworkPluginEndpointType, func(name string, client *plugins.Client) {
-		c := driverapi.Capability{
-			Scope: driverapi.GlobalScope,
+		// negotiate driver capability with client
+		d := newDriver(name, client)
+		var capResp api.GetCapabilityResponse
+		if err := d.(*driver).call("GetCapability", nil, &capResp); err != nil {
+			log.Errorf("error getting capability for %s due to %v", name, err)
```

Comment:
Yeah, I noticed this. 
On one hand, we have changed the plugin's registration process, and I think the existing plugin should make some relevant modifications to adapt this; one the other hand, I'm totally agreed with you about the compatibility. 
So if you think to default the scope to be `GlobalScope` is better, I'm totally OK with that. :)